raiden/app.py

```diff
@@ -130,7 +134,7 @@ def start(self):
         """ Start the raiden app. """
         self.raiden.start()
 
-    def stop(self, leave_channels: bool = False):
+    def stop(self, leave_channels: bool=False):
```

Comment:
Same as above

Reply:
Humm, I think i have a precommit-hook which removes this, gracias

Reply:
Yeah that is correct without annotations but with they are 'required'